runtime/parachains/src/paras_inherent/mod.rs

```diff
@@ -900,94 +941,41 @@ fn compute_entropy<T: Config>(parent_hash: T::Hash) -> [u8; 32] {
 /// 2. If exceeded:
 ///   1. Check validity of all dispute statements sequentially
 /// 2. If not exceeded:
-///   1. Sort the disputes based on locality and age, locality first.
-///   1. Split the array
-///   1. Prefer local ones over remote disputes
```

Comment:
Why do we no longer prioritize local disputes? It could also be done without randomness

Reply:
As far as I understood the dispute set is sorted (local before remote) so they are still prioritized.